components/channel_invite_modal/index.ts

```diff
@@ -59,7 +61,7 @@ function makeMapStateToProps(initialState: GlobalState, initialProps: OwnProps)
             profilesInCurrentChannel = getProfilesInCurrentChannel(state) as UserProfileValue[];
             profilesNotInCurrentTeam = getProfilesNotInCurrentTeam(state) as UserProfileValue[];
         }
-
+        const profilesFromRecentDMs = getRecentProfilesFromDMs(state) as UserProfileValue[];
```

Comment:
I'm not sure why the other lines above this have an `as UserProfileValue[]` at the end, but I don't think we should need it here

Reply:
I'm also not sure why other lines have `as UserProfileValue[]`. For `getRecentProfilesFromDMs(state)`, it's not required. I'll remove it.

Reply:
I think variable definitions , forces to use` as UserProfileValue[]`
```ts
 let profilesNotInCurrentChannel: UserProfileValue[];
 let profilesInCurrentChannel: UserProfileValue[];
 let profilesNotInCurrentTeam: UserProfileValue[];
```